tests/insert-pragma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,122 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`pragma-alreadyinserted.php 1`] = `
+<?php
+/**
+ * This already has the pragma, it shouldn't get another.
+ *
+ * @format
+ * @author    Bob Bobberson
+ * @copyright Doge Inc. 2018 - All Rights Reserved
+ */
+
+echo "Hello World";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+/**
+ * This already has the pragma, it shouldn't get another.
+ *
+ * @format
+ * @author    Bob Bobberson
+ * @copyright Doge Inc. 2018 - All Rights Reserved
+ */
+
+echo "Hello World";
+
+`;
+
+exports[`pragma-nodocblock.php 1`] = `
+<div>
+  inline stuff up top
+</div>
+
+<?php
+
+echo "Hello World";
+
+// non-docblock comment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  inline stuff up top
+</div>
+
+<?php /** @format */
```

Comment:
Not sure it is good idea do this, maybe better insert new php tag at the top of file with pragma, but we can do this after solve problem with inline printing

Reply:
Agree, I wasn't thrilled with doing it this way

Reply:
I think I will just drop `--insert-pragma` support for files that don't start with a php tag.  Seems simplest.